### PR TITLE
Update modal's `show` method to accept `relatedTarget` as an argument

### PR DESCRIPTION
**Pull request by WinterSilence**
Add new example for `bootstrap.Modal.show(relatedTarget)`

### CODE REVIEW

**Comment by XhmikosR** [comment]
What exactly are you trying to improve here?

**Comment by WinterSilence** [comment]
@XhmikosR explore missed parameter

**Comment by XhmikosR** [comment]
@rohit2sharma95 please check this out if it's right or now (and feel free to rename the PR title/variables etc )

File: site/content/docs/5.0/components/modal.md [hunk 1]
```diff
@@ -892,6 +892,13 @@ Manually opens a modal. **Returns to the caller before the modal has actually be
 myModal.show()
 ```
 
+Also, you can pass related target:
```
Comment (rohit2sharma95) [inline site/content/docs/5.0/components/modal.md:895]: Also, please mention that the DOM element passed as the first argument can be received in the modal events (as the `relatedTarget` property)

**Review by rohit2sharma95 — APPROVED** [review]
Thanks @WinterSilence

**Review by GeoSot — APPROVED** [review]

**Pull request merged**